src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -118,9 +118,13 @@ private void CompositeGUI()
 
                 EditorGUILayout.BeginHorizontal("Box");
                 {
-                    string[] compositionOptions = new string[] { "Final composited texture", "Quadrant with intermediate textures" };
-                    GUIContent renderingModeLabel = new GUIContent("Display texture", "Choose between displaying the composited video texture or seeing intermediate textures displayed in 4 sections (bottom left: input video, top left: opaque hologram, top right: hologram alpha mask, bottom right: hologram alpha-blended onto video)");
+                    string[] compositionOptions = new string[] { "Final composite", "Intermediate textures" };
+                    GUIContent renderingModeLabel = new GUIContent("Preview display", "Choose between displaying the composited video texture or seeing intermediate textures displayed in 4 sections (bottom left: input video, top left: opaque hologram, top right: hologram alpha mask, bottom right: hologram alpha-blended onto video)");
                     textureRenderMode = EditorGUILayout.Popup(renderingModeLabel, textureRenderMode, compositionOptions);
+                    if (compositionManager != null && compositionManager.TextureManager != null)
+                    {
+                        compositionManager.TextureManager.IsQuadrantVideoOutputRequired = textureRenderMode == (int)VideoRecordingFrameLayout.Quad;
+                    }
```

Comment:
should we consider another flag on the composition manager called VideoOutputLayout, similar to VideoRecordingLayout?

Reply:
Right now the video output layout is always "Composite" because the BlackMagic 4K capture card does not support outputting 4K unless the input video is 4K, and I don't think we want to output 1080p quadrant video (each sub-frame would be only 720p).

---
In reply to: [310675484](https://github.com/microsoft/MixedReality-SpectatorView/pull/172#discussion_r310675484) [](ancestors = 310675484)